addons/hr_recruitment/models/hr_job.py

```diff
@@ -59,8 +59,9 @@ def _get_default_favorite_user_ids(self):
     interviewer_ids = fields.Many2many('res.users', string='Interviewers', domain="[('share', '=', False), ('company_ids', 'in', company_id)]", tracking=True, help="The Interviewers set on the job position can see all Applicants in it. They have access to the information, the attachments, the meeting management and they can refuse him. You don't need to have Recruitment rights to be set as an interviewer.")
     extended_interviewer_ids = fields.Many2many('res.users', 'hr_job_extended_interviewer_res_users', compute='_compute_extended_interviewer_ids', store=True)
     industry_id = fields.Many2one('res.partner.industry', 'Industry', tracking=True)
-    date_from = fields.Date(help="Is set, update candidates availability once hired for that specific mission.")
+    date_from = fields.Date(help="Is set, update applicants availability once hired for that specific mission.")
     date_to = fields.Date()
+    compensation = fields.Float()
```

Comment:
I suppose this field should be Monetary ?

```suggestion
    compensation = fields.Monetary()
```

Reply:
Yes that makes sense, though it also required me to add a currency_id field.